components/carousel/carousel.scss

```diff
@@ -0,0 +1,86 @@
+.ui-carousel {
+  position: relative;
+  overflow: hidden;
+}
+
+.ui-carousel-track {
+  display: flex;
+  position: relative;
+  transition: var(--tx-carousel-track-transition);
+}
+
+.ui-carousel-item {
+  flex: 1 0 100%;
+  position: relative;
+  min-height: var(--tx-carousel-item-min-height);
+}
+
+.ui-carousel-item-image {
```

Comment:
what about rule "styles sorting" for ui kit? Should we do it by alphabetical or type groups?
@mAiNiNfEcTiOn  @asci